templates/docs/examples/patterns/tables/_script-expanding.js

```diff
@@ -0,0 +1,47 @@
+/**
+  Toggles the necessary aria- attributes' values on the table panels
+  to show or hide them.
+  @param {HTMLElement} element The tab that acts as the handles.
+  @param {Boolean} show Whether to show or hide the expanded row panel.
+*/
+function toggleExpanded(element, show) {
+  var target = document.getElementById(element.getAttribute('aria-controls'));
+
+  if (target) {
+    element.setAttribute('aria-expanded', show);
```

Comment:
Should we also user `aria-pressed` here? @lyubomir-popov ?

Reply:
I don't think you can use two states. You have to pick one and expanded makes more sense to me:
https://www.w3.org/TR/wai-aria/#button
![image](https://user-images.githubusercontent.com/2741678/109688474-35943a80-7b7c-11eb-9ba1-7d9072d62ea3.png)

Reply:
It doesn't seem to be mentioned anywhere that states are exclusive. I think the button can have multiple states if they are applicable - it can be `aria-expanded` and `aria-pressed` at the same time, or `aria-busy`. These seem to be independent states that are turned on or off.

On a side note of that - maybe we should style buttons with `aria-expanded` the same way as we style `aria-pressed` to mark they are active? These would save us adding second `pressed` state when `expanded` is already on the button.

Reply:
> maybe we should style buttons with `aria-expanded` the same way as we style `aria-pressed` to mark they are active

yes, definitely!